styles/item.less

```diff
@@ -3,19 +3,20 @@
   align-items: center;
   justify-content: space-between;
   margin: 0 0 10px 0;
-  padding: 0 20px;
+  padding: 10px 20px;
```

Comment:
For the long text being wrapper, it gets glued tied to the borders. the 10px is there to make a distinction between the text and the borders. what do you think?